versions.txt

```diff
@@ -1,2 +1,5 @@
 # List of packages to bump:
 # Format: [package_name]@[new_version]
+
+boss@4.2.0
```

Comment:
These should never be added when merging to master, as changes to master will be released in the same day. So please make sure to revert all changes to this file.

Reply:
Happy to revert, but how do I make sure we get minor bumps instead of just patch fixes? Do I edit the package.json versions manually?

Reply:
See the contributing docs... https://github.com/tinymce/tinymce/blob/develop/CONTRIBUTING.md#publishing-process. So in this case you'd run `yarn lerna version` after this has been merged to do the minor version bumps.